css/compositing/mix-blend-mode/META.yml

```diff
@@ -22,3 +22,17 @@ links:
           status: FAIL
         - test: mix-blend-mode-mask.html
           status: FAIL
+        - test: mix-blend-mode-blended-element-overflow-hidden-and-border-radius.html
```

Comment:
It looks like we also could have gotten these from TestExpectations, I found https://chromium.googlesource.com/chromium/src.git/+/5b7ecdb7b265a278367d9334656531c69bd4089b%5E%21/#F0 via the bug. But the bug is right.